aptos-move/mvhashmap/src/versioned_delayed_fields.rs

```diff
@@ -101,13 +101,13 @@ impl<K: Copy + Clone + Debug + Eq> VersionedValue<K> {
 
         match self.versioned_map.entry(txn_idx) {
             Entry::Occupied(mut o) => {
-                let bypass = match &**o.get() {
+                let bypass = match &***o.get() {
```

Comment:
this is not very readable, can we do as_ref().deref() instead?

Reply:
@zekun000 CachePadded is another 20% of memory, not sure if it is useful at all. if it isn't I can remove it and rest of the file doesn't change (basically replace CachePadded with Box)

Reply:
not sure without actually benchmarking it, cc @gelash if you have any ideas